vita3k/main.cpp

```diff
@@ -86,12 +86,19 @@ static void run_execv(char *argv[], EmuEnvState &emuenv) {
 int main(int argc, char *argv[]) {
     ZoneScoped; // Tracy - Track main function scope
     Root root_paths;
-    root_paths.set_base_path(string_utils::utf_to_wide(SDL_GetBasePath()));
-    root_paths.set_pref_path(string_utils::utf_to_wide(SDL_GetPrefPath(org_name, app_name)));
+
+    app::init_paths(root_paths);
 
     if (logging::init(root_paths, true) != Success)
         return InitConfigFailed;
 
+    LOG_INFO("Base path: {}", root_paths.get_base_path_string());
+    LOG_INFO("Shared assets path: {}", root_paths.get_shared_path_string());
+    LOG_INFO("Log path: {}", root_paths.get_log_path_string());
+    LOG_INFO("User config path: {}", root_paths.get_config_path_string());
+    LOG_INFO("User pref path: {}", root_paths.get_pref_path_string());
+    LOG_INFO("User cache path: {}", root_paths.get_cache_path_string());
+
```

Comment:
This actually doesn't print the proper pref path (or config path) if it was changed in the config. These logs should be put after config::init_config for the paths to be always correct.
I would suggest putting this in app::init at the same time you copy the paths to the emuenv structure.